uasurfer.go

```diff
@@ -67,6 +67,8 @@ const (
 	BrowserYandex        BrowserID = 19
 	BrowserCocCoc        BrowserID = 20
 	BrowserBot           BrowserID = 21 // Bot list begins here
+	BrowserMSOutlook     BrowserID = 22
```

Comment:
can you please check that there is no >= BrowserBot check anywhere, otherwise good